src/components/home/Team/index.js

```diff
@@ -0,0 +1,75 @@
+import _ from "lodash"
+import Img from "gatsby-image"
+import React from "react"
+
+import { StaticQuery, graphql } from "gatsby"
+
+import "./index.module.css"
+
+const socialLink = (platform, username) => {
+  switch (platform) {
+    case "twitter":
+      return `https://twitter.com/${username}`
+    default:
+      return "https://subvisual.co"
+  }
+}
+
+const Team = props => {
+  return (
+    <ul styleName="root">
+      {props.members.map(({ name, role, social, photo }) => (
+        <li styleName="member" key={name}>
+          <Img styleName="photo" fluid={photo.childImageSharp.fluid} />
+          <div styleName="name">
+            <span styleName="bold">{name}</span> <br />
+            {role}
+          </div>
+          <ul styleName="links">
+            {_.map(social, (name, platform) => (
+              <li styleName="link" key={platform}>
+                <a target="_blank" href={socialLink(platform, name)}>
```

Comment:
Shouldn't this be using Gatsby `Link` component? Honest question, have no idea the tradeoff here, thought it was the convention.

Reply:
That's just for internal navigation.